index.py

```diff
@@ -20,7 +23,38 @@ def __init__(self, args):
         mqtt.connect(not args.offline)
         self.modules = None
         self.states = {}
-
+    def handle_conn(self,conn,addr):
+        try:
+            while True:
+                conn.sendall(f"enter target game, options are {list(self.states.keys())}:".encode())
+                response = conn.recv(1024).decode().strip()
+                if response in self.states:
+                    break
+                conn.sendall(f'{response} is not a game\n\r'.encode())
+        except Exception:
+            raise
+        #print(f"got {response} client")
+        hasher = hashlib.sha256()
+        hasher.update((addr[0]+str(addr[1])).encode())
```

Comment:
Dit gaat stuk als mensen achter een NAT device zitten, mss ook de client port meenemen in de hash